charts/chronicle/templates/statefulset.yaml

```diff
@@ -125,11 +125,17 @@ spec:
                   --interface 0.0.0.0:{{ .Values.port}} \
                   {{- if .Values.auth.required }}
                   --require-auth \
+                  {{- end }}
+                  {{- if .Values.auth.id.claims }}
```

Comment:
you have a default value for this  so it is always going to be true unless the user explicitly sets it to nil

Reply:
Default is gone. Chronicle provides `[iss, sub]` by default.